modules/virt-about-containerdisks-for-vms.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-about-containerdisks-for-vms_{context}"]
+= About using containerDisk volumes for virtual machines
+
+A container disk is a virtual machine image stored as a container in an
+image registry. You can attach a container disk as an ephemeral containerDisk volume to
+a virtual machine by using the virtual machine wizard or by adding it to a
+virtual machine manifest.
+
+Container disks are not limited to a single virtual machine and can be used
```

Comment:
`Container disks are not limited to a single virtual machine` might lead to the wrong impression, instead:
`Container disks are useful to deliver the same disk images to multiple virtual machines`